spec/spec.md

```diff
@@ -580,48 +558,47 @@ where:
 - `ERROR_TOO_MANY_PAYLOADS` if the wallet endpoint is unable to sign all transactions due to exceeding implementation limits. These limits may be available via [`get_capabilities`](#get_capabilities).
 
 ##### Description
 
-The wallet endpoint should attempt to simulate the transactions provided by data and present them to the user for approval (if applicable). If approved (or if it does not require approval), the wallet endpoint should sign the transactions with the private key for the authorized account, and return the signed transactions to the dapp endpoint.
+The wallet endpoint should attempt to simulate the transactions provided by data and present them to the user for approval (if applicable). If approved (or if it does not require approval), the wallet endpoint should sign the transactions with the private keys for the requested authorized account addresses, and return the signed transactions to the dapp endpoint.
 
-#### sign_and_send_transaction
+#### sign_and_send_transactions
 
 ##### JSON-RPC method specification
 
 ###### Method
 {: .no_toc }
 
 ```
-sign_and_send_transaction
+sign_and_send_transactions
 ```
 
 ###### Params
 {: .no_toc }
 
 ```
 {
-    “auth_token”: “<auth_token>”,
     “payloads”: [“<transaction>”, ...],
-    “commitment”: “<commitment_level>”,
-    "cluster": "<cluster>",
-    "skip_preflight": <skip_preflight>,
-    "preflight_commitment": "<preflight_commitment>",
+    "options": {
+        “commitment”: “<commitment_level>”,
+        "skip_preflight": <skip_preflight>,
+        "preflight_commitment": "<commitment_level>",
+    }
 }
 ```
 
 where:
 
-- `auth_token`: an auth_token returned by [`authorize`](#authorize), [`reauthorize`](#reauthorize), or [`clone_authorization`](#clone_authorization) for which access to `sign_and_send_transaction` was requested
-- `transaction`: one or more base64-encoded transaction payload to sign, each with a maximum pre-encoding size of `MAX_TRANSACTION_SZ`
-- `commitment_level`: one of `processed`, `confirmed`, or `finalized`
-- `cluster`: (optional) if set, the Solana network cluster to send the signed transaction(s) to; supported values include `mainnet_beta`, `testnet`, `devnet`. If not set, the wallet endpoint determines which cluster to send transactions to.
-- `skip_preflight`: (optional) see the JSON-RPC [sendTransaction](https://docs.solana.com/developing/clients/jsonrpc-api#sendtransaction) API. If not set, defaults to `false`.
-- `preflight_commitment`: (optional) see the JSON-RPC [sendTransaction](https://docs.solana.com/developing/clients/jsonrpc-api#sendtransaction) API. If not set, defaults to `finalized`.
+- `payloads`: one or more base64-encoded transaction payload to sign
+- `options`: a JSON object, containing:
+  - `commitment`: (optional) if set, one of `processed`, `confirmed`, or `finalized`, specifying the desired commitment level that must be reached for this request to be successful. If not set, defaults to `confirmed`.
```

Comment:
So, hold on, I've been meaning to ask this. Are we actually making the wallets do transaction confirmation? This would be a new thing, for wallet adapters.

Take, for instance, the current Phantom wallet adapter for desktop web:

https://github.com/solana-labs/wallet-adapter/blob/master/packages/wallets/phantom/src/adapter.ts#L209-L210

It sends the transaction, returns the (unasserted) signature, and then leaves confirmation up to the dApp.

* Control returns to the dApp right away
* Users can perform new orthogonal transactions while waiting for the last one(s) to complete
* The dApp can care about the confirmation or not (eg. maybe it's no longer important if the user has navigated away from that screen)